litestar/utils/helpers.py

```diff
@@ -2,7 +2,7 @@
 
 from dataclasses import dataclass
 from enum import Enum
-from typing import TYPE_CHECKING, Any, Generic, TypeVar, cast
+from typing import TYPE_CHECKING, Any, Container, Generic, TypeVar, cast
```

Comment:
IIRC, `typing.Container` is deprecated since 3.9
```suggestion
from collections.abc import Container
```